Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine:3.12
+FROM alpine:latest
```

Comment:
I think the risk with this is that the build starts breaking due to unexpected changes in Alpine. I'm okay with that risk, though, because we should still be able to use the last successful build while fixing the broken one, so it shouldn't make much of a mess.

Reply:
I was thinking similarly.